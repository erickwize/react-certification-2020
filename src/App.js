import logo from './logo.svg';
import './App.css';
import styled from "styled-components"
import Header from './components/header/header.jsx';
import VideoCard from "./components/videocard/videocard.jsx"
//import "./css/index.css"
import 'bootstrap/dist/css/bootstrap.min.css';
import js from "../src/data/yt";
import { Container } from 'react-bootstrap';




const VideoList = js.items.map((vid)=>(
  <div key={vid.id.kind} className="cards">
    <VideoCard 
    thumbnail={vid.snippet.thumbnails.high.url} 
    title={vid.snippet.title} url={vid.id.videoId}/>
  </div>
));


function App() {
  return (
    <div>
      <div className="App">
        <Header/>
    </div>
        {VideoList}
    </div>
    
  );
}

export default App;
