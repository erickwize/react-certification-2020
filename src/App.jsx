import React, { Fragment, useState } from "react";
import Header from "./components/Header";
import VideoList from "./components/VideoList";
import PlayerView from "./components/PlayerView";
import Spinner from "./components/layout/Spinner";
import useApiSearch from "./hooks/useApiSearch";
import styled, { createGlobalStyle } from "styled-components";

const GlobalStyle = createGlobalStyle`
  * {
    box-sizing: border-box;
    margin: 0;
    padding: 0;
  }
`;

const MainTitle = styled.h1`
  font-family: "Oswald", sans-serif;
  font-size: 48px;
  text-align: center;
  margin: 90px auto 35px;
`;

const Content = styled.div`
  padding: 0 30px;
  margin: 1px solid blue;
`;

const App = () => {
  const [searchText, setSearchText] = useState("Wizeline Academy");
  const [selectedVideo, setSelectedVideo] = useState({});
  const [shouldShowSearch, setShouldShowSearch] = useState(true);
  const [searchResults, relatedVideos, loading] = useApiSearch(
    searchText,
    selectedVideo
  );

  const handleSearch = (text) => {
    setSearchText(text);
    setShouldShowSearch(true);
  };

  const handleSelectedVideo = (videoProps) => {
    setSelectedVideo(videoProps);
    setShouldShowSearch(false);
  };

  return (
    <Fragment>
      <GlobalStyle />
      <div className="App">
        <Header handleSearch={handleSearch} />
        {loading ? (
          <Spinner />
        ) : (
          <Content>
            {shouldShowSearch ? (
              <Fragment>
                <MainTitle>Welcome to YouTubit</MainTitle>
                <VideoList
                  videos={searchResults}
                  handleSelectedVideo={handleSelectedVideo}
                />
              </Fragment>
            ) : (
              <PlayerView
                selectedVideo={selectedVideo}
                relatedVideos={relatedVideos}
                handleSelectedVideo={handleSelectedVideo}
              />
            )}
          </Content>
        )}
      </div>
    </Fragment>
  );
};

export default App;
