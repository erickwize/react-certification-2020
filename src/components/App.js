import React from 'react'
import { Route, Switch } from 'react-router'
import { BrowserRouter as Router } from 'react-router-dom'
import { createBrowserHistory } from 'history'
import { ThemeContextProvider } from '../context/ThemeContext'
import { YoutubeContextProvider } from '../context/YoutubeContext'
import Layout from './Layout'
import Home from '../pages/Home'
import GlobalStyle from '../globalStyles'
import VideoDetail from '../pages/VideoDetail'

const customHistory = createBrowserHistory()
const App = () => {
	return (
		<YoutubeContextProvider>
			<ThemeContextProvider>
				<GlobalStyle />
				<Router history={customHistory}>
					<Switch>
						<Route exact path="/">
							<Layout>
								<Home />
							</Layout>
						</Route>
						<Route path="/video-detail/:videoId">
							<Layout>
								<VideoDetail />
							</Layout>
						</Route>
					</Switch>
				</Router>
			</ThemeContextProvider>
		</YoutubeContextProvider>
	)
}

export default App
