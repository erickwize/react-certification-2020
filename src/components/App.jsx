/* global gapi*/
/*eslint no-undef: "error"*/

import React, { Fragment, useState, useEffect } from "react";
import Header from "./Header";
import Videos from "./Videos";
import styled, { createGlobalStyle } from "styled-components";
// import data from "../mock/youtube-videos-mock.json";

const GlobalStyle = createGlobalStyle`
  * {
    box-sizing: border-box;
    margin: 0;
    padding: 0;
  }
`;

const Content = styled.main`
  padding: 30px;

  .mainTitle {
    font-family: "Oswald", sans-serif;
    font-size: 48px;
    text-align: center;
    margin: 60px auto 35px;
  }
`;

const App = () => {
  const [searchText, setSearchText] = useState("Wizeline");
  const [fetchedVideos, setFetchedVideos] = useState([]);

  // eslint-disable-next-line no-undef
  const apiKey = process.env.REACT_APP_API_KEY;
  let discovery = {};

  const handleClientLoad = async () => {
    try {
      const response = await fetch(
        "https://www.googleapis.com/discovery/v1/apis/youtube/v3/rest"
      );
      discovery = await response.json();
      gapi.load("client", handleClientInit);
    } catch (error) {
      console.error("handleClientLoad: Something went wrong... ", error);
    }
  };

  const handleClientInit = async () => {
    try {
      await gapi.client.init({
        apiKey: apiKey,
        discoveryDocs: [discovery],
      });
    } catch (error) {
      console.error("handleClientInit: Something went wrong... ", error);
    }
  };

  const searchRequest = async (searchText) => {
    try {
      const response = await gapi.client.youtube.search.list({
        part: ["snippet"],
        maxResults: 24,
        q: searchText,
        type: ["video"],
      });
      setFetchedVideos(response.result.items);
    } catch (error) {
      console.error("searchRequest: Something went wrong... ", error);
    }
  };

  useEffect(() => {
    handleClientLoad();
  }, []);

  useEffect(() => {
    // fetch only when text is not empty && youtube api is loaded
    searchRequest(searchText);
  }, [searchText]);

  const handleSearch = (text) => {
    setSearchText(text);
  };

  console.log("App fetched: ", fetchedVideos);

  return (
    <Fragment>
      <GlobalStyle />
      <div className="App">
        <Header handleSearch={handleSearch} />
        <Content>
          <h1 className="mainTitle">Welcome to YouTubit</h1>
          <Videos fetchedVideos={fetchedVideos} />
        </Content>
      </div>
    </Fragment>
  );
};

export default App;
