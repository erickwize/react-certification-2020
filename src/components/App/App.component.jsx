import React from 'react';
import { BrowserRouter } from 'react-router-dom';


//Components
import Layout from '../Layout';
import { Header } from '../../components/Header/Header.component';
import GlobalProvider from '../../providers/Global.provider';
import { Menu } from '../Menu/Menu.component';
import { Routes } from '../Routes/Routes.component';

function App() {

  return (
    <BrowserRouter>
      <GlobalProvider>
        <Menu/>
        <Header/>
        <Layout>
          <Routes/>
        </Layout>
      </GlobalProvider>
    </BrowserRouter>
  );
}

export default App;
