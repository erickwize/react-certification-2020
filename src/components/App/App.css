html {
  font-size: 1.125rem;
  line-height: 1.6;
  font-weight: 400;
  font-family: sans-serif;
  box-sizing: border-box;
  scroll-behavior: smooth;
  -webkit-tap-highlight-color: rgba(0, 0, 0, 0);
}

* {
  box-sizing: inherit;
}

body {
  margin: 0;
  padding: 0;
  text-rendering: optimizeLegibility;
}

a {
  text-decoration: none;
  font-weight: bold;
  color: white;
}

a:active {
  color: blueviolet;
}
