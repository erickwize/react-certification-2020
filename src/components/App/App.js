import React from 'react'
import { BrowserRouter, Switch, Route } from 'react-router-dom'
import { ContextProvider } from '../../context/context'
import AuthProvider from '../../providers/Auth'
import Home from '../../pages/Home/Home.page'
import SecretPage from '../../pages/Secret'
import Private from '../Private/Private'
import Layout from '../Layout/Layout'

const App = React.memo(() => (
  <BrowserRouter>
    <AuthProvider>
      <ContextProvider>
        <Layout>
          <Switch>
            <Route exact path="/">
              <Home />
            </Route>
            <Route exact path="/login">
              <Home />
            </Route>
            <Private exact path="/secret">
              <SecretPage />
            </Private>
            <Route path="*">
              <Home />
            </Route>
          </Switch>
        </Layout>
      </ContextProvider>
    </AuthProvider>
  </BrowserRouter>
))

export default App
