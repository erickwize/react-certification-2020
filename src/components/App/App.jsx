import React from 'react';
import { HashRouter as BrowserRouter, Switch, Route } from 'react-router-dom';

import AuthProvider from '../../providers/Auth/Auth.provider';
import HomePage from '../../pages/Home/Home';
import NotFound from '../../pages/NotFound/NotFound';
import FavoritePage from '../../pages/Favorite/Favorite';
import FavoriteVideoPage from '../../pages/FavoriteVideo/FavoriteVideo';
import VideoPage from '../../pages/Video/Video';
import Private from '../Private/Private';
import Header from '../Header/Header';
import { useSelector } from 'react-redux';


function App() {
  const themeReducer = useSelector((state) => state.themeReducer);
  
  const themeStyle = `
    background-color: ${ themeReducer.mode == 'dark' ? '#303030;' : 'white;' }
    color: ${ themeReducer.mode == 'dark' ? '#fff' : '#1f1f1f' };
  `;

  const { body } = document;
  body.style = themeStyle;

  return (
    
    <BrowserRouter>   
    <AuthProvider>         
      <Header />       
          <Switch>
            <Route exact path="/">
              <HomePage />
            </Route>
            <Private exact path="/favorites">
              <FavoritePage theme={themeReducer.mode} />
            </Private>
            <Private exact path="/favorites/:id">
              <FavoriteVideoPage theme={themeReducer.mode} />
            </Private>
            <Route path="/:id">
              <VideoPage />
            </Route>            
            <Route path="*">
              <NotFound />
            </Route>
          </Switch>
      </AuthProvider>
    </BrowserRouter>
  
  );
}

export default App;