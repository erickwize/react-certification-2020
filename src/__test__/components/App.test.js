import { render } from '@testing-library/react'
import React from 'react'
import ReactDOM from 'react-dom'
import App from '../../components/App'

describe('App testing', () => {
	it('Renders without crashing', () => {
		const div = document.createElement('div')
		ReactDOM.render(<App />, div)
	})

	it('App Snapshot', () => {
		const component = render(<App />)
		expect(component.container).toMatchSnapshot()
	})
})
