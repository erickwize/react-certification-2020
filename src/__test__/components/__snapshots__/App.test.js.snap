// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App testing App Snapshot 1`] = `
<div>
  <header
    class="sc-bdfBwQ gJokXU"
  >
    <div
      class="sc-dlfnbm bhsxjX"
    >
      <button
        class="sc-gsTCUz lndufi"
      >
        <img
          alt="menu"
          src="menu.svg"
        />
      </button>
      <input
        class="sc-hKgILt bERkee"
        placeholder="Search"
        type="text"
      />
    </div>
    <div
      class="sc-eCssSg kiYOCg"
    >
      <label
        class="sc-gKsewC kzdNfg"
      >
        <input
          type="checkbox"
        />
        <span />
      </label>
      <div
        class="sc-jSgupP dcwTA-d"
      >
        <img
          alt="avatar"
          src="user.svg"
        />
      </div>
    </div>
  </header>
  <main>
    <div
      class="sc-crrsfI fTxdms"
    >
      <h1
        class="sc-dQppl jZASLv"
      >
        Videos List
      </h1>
      <div
        class="sc-iqHYGH iKAPlC"
      >
        <div
          class="sc-iBPRYJ dmMilx"
        >
          <img
            alt="thumbnail"
            class="sc-fubCfw fKSrkK"
            src="https://yt3.ggpht.com/ytc/AAUvwnighSReQlmHl_S_vSfvnWBAG5Cw4A0YxtE0tm5OpQ=s88-c-k-c0xffffffff-no-rj-mo"
          />
          <div
            class="sc-pFZIQ"
          >
            <span
              class="sc-jrAGrp gZeKgx"
            >
              Wizeline
            </span>
            <p
              class="sc-kEjbxe hAIuZE"
            >
              Wizeline transforms how teams build technology. Its customers accelerate the delivery of innovative products with proven solutions, which combine Wizeline's ...
            </p>
          </div>
        </div>
        <div
          class="sc-iBPRYJ dmMilx"
        >
          <img
            alt="thumbnail"
            class="sc-fubCfw fKSrkK"
            src="https://i.ytimg.com/vi/nmXMgqjQzls/default.jpg"
          />
          <div
            class="sc-pFZIQ"
          >
            <span
              class="sc-jrAGrp gZeKgx"
            >
              Video Tour | Welcome to Wizeline Guadalajara
            </span>
            <p
              class="sc-kEjbxe hAIuZE"
            >
              Follow Hector Padilla, Wizeline Director of Engineering, for a lively tour of our office. In 2018, Wizeline opened its stunning new office in Guadalajara, Jalisco, ...
            </p>
          </div>
        </div>
        <div
          class="sc-iBPRYJ dmMilx"
        >
          <img
            alt="thumbnail"
            class="sc-fubCfw fKSrkK"
            src="https://i.ytimg.com/vi/HYyRZiwBWc8/default.jpg"
          />
          <div
            class="sc-pFZIQ"
          >
            <span
              class="sc-jrAGrp gZeKgx"
            >
              Wizeline Guadalajara | Bringing Silicon Valley to Mexico
            </span>
            <p
              class="sc-kEjbxe hAIuZE"
            >
              Wizeline continues to offer a Silicon Valley culture in burgeoning innovation hubs like Mexico and Vietnam. In 2018, our Guadalajara team moved into a ...
            </p>
          </div>
        </div>
        <div
          class="sc-iBPRYJ dmMilx"
        >
          <img
            alt="thumbnail"
            class="sc-fubCfw fKSrkK"
            src="https://i.ytimg.com/vi/Po3VwR_NNGk/default.jpg"
          />
          <div
            class="sc-pFZIQ"
          >
            <span
              class="sc-jrAGrp gZeKgx"
            >
              Wizeline hace sentir a empleados como en casa
            </span>
            <p
              class="sc-kEjbxe hAIuZE"
            >
              En el 2014, Bismarck fundó Wizeline, compañía tecnológica que trabaja con los corporativos ofreciendo una plataforma para que desarrollen software de forma ...
            </p>
          </div>
        </div>
        <div
          class="sc-iBPRYJ dmMilx"
        >
          <img
            alt="thumbnail"
            class="sc-fubCfw fKSrkK"
            src="https://i.ytimg.com/vi/7PtYNO6g7eI/default.jpg"
          />
          <div
            class="sc-pFZIQ"
          >
            <span
              class="sc-jrAGrp gZeKgx"
            >
              We Are Wizeline
            </span>
            <p
              class="sc-kEjbxe hAIuZE"
            >
              Engineering a better tomorrow. Wizeline is a global software development company that helps its clients solve their biggest challenges with design and ...
            </p>
          </div>
        </div>
        <div
          class="sc-iBPRYJ dmMilx"
        >
          <img
            alt="thumbnail"
            class="sc-fubCfw fKSrkK"
            src="https://i.ytimg.com/vi/YuW0CE_8i1I/default.jpg"
          />
          <div
            class="sc-pFZIQ"
          >
            <span
              class="sc-jrAGrp gZeKgx"
            >
              Wizeline Thrives in Mexico City
            </span>
            <p
              class="sc-kEjbxe hAIuZE"
            >
              Our vibrant Mexico City office is home to agile software engineers, talented UX designers, and brilliant data scientists. Learn about the rich history of Mexico City.
            </p>
          </div>
        </div>
        <div
          class="sc-iBPRYJ dmMilx"
        >
          <img
            alt="thumbnail"
            class="sc-fubCfw fKSrkK"
            src="https://i.ytimg.com/vi/CHzlSGRvWPs/default.jpg"
          />
          <div
            class="sc-pFZIQ"
          >
            <span
              class="sc-jrAGrp gZeKgx"
            >
              Wizeline
            </span>
            <p
              class="sc-kEjbxe hAIuZE"
            >
              El plan de Wizeline, una empresa de inteligencia artificial, para ayudar a crecer la comunidad de ciencia de datos en CDMX y todo el país, a través de cursos ...
            </p>
          </div>
        </div>
        <div
          class="sc-iBPRYJ dmMilx"
        >
          <img
            alt="thumbnail"
            class="sc-fubCfw fKSrkK"
            src="https://i.ytimg.com/vi/cjO2fJy8asM/default.jpg"
          />
          <div
            class="sc-pFZIQ"
          >
            <span
              class="sc-jrAGrp gZeKgx"
            >
              A Day in the Life of an Engineering Manager at Wizeline
            </span>
            <p
              class="sc-kEjbxe hAIuZE"
            >
              Fernando Espinoza shares his experience working as an engineering manager at Wizeline and mentoring other engineers. Learn about Fernando's passions ...
            </p>
          </div>
        </div>
        <div
          class="sc-iBPRYJ dmMilx"
        >
          <img
            alt="thumbnail"
            class="sc-fubCfw fKSrkK"
            src="https://i.ytimg.com/vi/zClI9OjgKXM/default.jpg"
          />
          <div
            class="sc-pFZIQ"
          >
            <span
              class="sc-jrAGrp gZeKgx"
            >
              Wizeline Technical Writing Academy | Featuring Eduardo Ocejo
            </span>
            <p
              class="sc-kEjbxe hAIuZE"
            />
          </div>
        </div>
        <div
          class="sc-iBPRYJ dmMilx"
        >
          <img
            alt="thumbnail"
            class="sc-fubCfw fKSrkK"
            src="https://i.ytimg.com/vi/8bz9R61oY5o/default.jpg"
          />
          <div
            class="sc-pFZIQ"
          >
            <span
              class="sc-jrAGrp gZeKgx"
            >
              Silicon Valley en México
            </span>
            <p
              class="sc-kEjbxe hAIuZE"
            >
              Empresas de Silicon Valley buscan establecerse en México por el gran talento que hay en nuestro país. Es una investigación de Roberto Domínguez.
            </p>
          </div>
        </div>
        <div
          class="sc-iBPRYJ dmMilx"
        >
          <img
            alt="thumbnail"
            class="sc-fubCfw fKSrkK"
            src="https://i.ytimg.com/vi/7dJFraOqcoQ/default.jpg"
          />
          <div
            class="sc-pFZIQ"
          >
            <span
              class="sc-jrAGrp gZeKgx"
            >
              Why Wizeline? featuring Juan Pablo Villa in Mexico City
            </span>
            <p
              class="sc-kEjbxe hAIuZE"
            >
              Juan Pablo, known as Gianpa at Wizeline, is a software engineer in our Mexico City office. Gianpa focuses on Android apps, is an integral part of our culture, ...
            </p>
          </div>
        </div>
        <div
          class="sc-iBPRYJ dmMilx"
        >
          <img
            alt="thumbnail"
            class="sc-fubCfw fKSrkK"
            src="https://i.ytimg.com/vi/w-Qwc_XJrWc/default.jpg"
          />
          <div
            class="sc-pFZIQ"
          >
            <span
              class="sc-jrAGrp gZeKgx"
            >
              Wizeline&#39;s 2020 Year in Review
            </span>
            <p
              class="sc-kEjbxe hAIuZE"
            >
              There's no doubt that 2020 has been an unprecedented year. However, amidst all the chaos, we achieved remarkable growth in various areas of our business.
            </p>
          </div>
        </div>
        <div
          class="sc-iBPRYJ dmMilx"
        >
          <img
            alt="thumbnail"
            class="sc-fubCfw fKSrkK"
            src="https://i.ytimg.com/vi/rjir_cHTl5w/default.jpg"
          />
          <div
            class="sc-pFZIQ"
          >
            <span
              class="sc-jrAGrp gZeKgx"
            >
              Why Wizeline? featuring Hugo Lopez in Mexico City
            </span>
            <p
              class="sc-kEjbxe hAIuZE"
            />
          </div>
        </div>
        <div
          class="sc-iBPRYJ dmMilx"
        >
          <img
            alt="thumbnail"
            class="sc-fubCfw fKSrkK"
            src="https://i.ytimg.com/vi/DcFK1x3NHGY/default.jpg"
          />
          <div
            class="sc-pFZIQ"
          >
            <span
              class="sc-jrAGrp gZeKgx"
            >
              Why Wizeline? (We&#39;re Hiring in Mexico!)
            </span>
            <p
              class="sc-kEjbxe hAIuZE"
            >
              A quick look at why people join Wizeline, what motivates us as a team and what it's like to work in our Guadalajara office. Learn more and apply here: ...
            </p>
          </div>
        </div>
        <div
          class="sc-iBPRYJ dmMilx"
        >
          <img
            alt="thumbnail"
            class="sc-fubCfw fKSrkK"
            src="https://i.ytimg.com/vi/3BzYWAqZgFw/default.jpg"
          />
          <div
            class="sc-pFZIQ"
          >
            <span
              class="sc-jrAGrp gZeKgx"
            >
              Why Wizeline? featuring Oswaldo Herrera in Mexico City
            </span>
            <p
              class="sc-kEjbxe hAIuZE"
            >
              Oswaldo is a software engineering in Wizeline's Mexico City office. He joined Wizeline because of the camaraderie and deep sense of commitment of our teams.
            </p>
          </div>
        </div>
        <div
          class="sc-iBPRYJ dmMilx"
        >
          <img
            alt="thumbnail"
            class="sc-fubCfw fKSrkK"
            src="https://i.ytimg.com/vi/3KVFmT-Tp2w/default.jpg"
          />
          <div
            class="sc-pFZIQ"
          >
            <span
              class="sc-jrAGrp gZeKgx"
            >
              Caso de Éxito AWS: Wizeline [Spanish]
            </span>
            <p
              class="sc-kEjbxe hAIuZE"
            >
              Central de socios de APN - https://amzn.to/2S7tIXM Fundada en 2014, Wizeline es una compañía joven e innovadora que nació en la nube para ofrecer soporte ...
            </p>
          </div>
        </div>
        <div
          class="sc-iBPRYJ dmMilx"
        >
          <img
            alt="thumbnail"
            class="sc-fubCfw fKSrkK"
            src="https://i.ytimg.com/vi/aKuPmY2m1Ro/default.jpg"
          />
          <div
            class="sc-pFZIQ"
          >
            <span
              class="sc-jrAGrp gZeKgx"
            >
              Wizeline&#39;s 2019 Year in Review
            </span>
            <p
              class="sc-kEjbxe hAIuZE"
            />
          </div>
        </div>
        <div
          class="sc-iBPRYJ dmMilx"
        >
          <img
            alt="thumbnail"
            class="sc-fubCfw fKSrkK"
            src="https://i.ytimg.com/vi/24sTHUyWhRM/default.jpg"
          />
          <div
            class="sc-pFZIQ"
          >
            <span
              class="sc-jrAGrp gZeKgx"
            >
              [1 of 2] Wizeline CEO shares career lessons from Google
            </span>
            <p
              class="sc-kEjbxe hAIuZE"
            >
              Founder & CEO Bismarck Lepe on growth opportunities at Wizeline and his career-path experience as an early Google employee. Join our team!
            </p>
          </div>
        </div>
        <div
          class="sc-iBPRYJ dmMilx"
        >
          <img
            alt="thumbnail"
            class="sc-fubCfw fKSrkK"
            src="https://i.ytimg.com/vi/IxGc1gSqB3A/default.jpg"
          />
          <div
            class="sc-pFZIQ"
          >
            <span
              class="sc-jrAGrp gZeKgx"
            >
              Wizeline Data Engineering featuring Tania Reyes
            </span>
            <p
              class="sc-kEjbxe hAIuZE"
            >
              Tania discovered her interest in Big Data while working at Wizeline and took Wizeline Academy courses to skill up and join the data team. Now, she works on ...
            </p>
          </div>
        </div>
        <div
          class="sc-iBPRYJ dmMilx"
        >
          <img
            alt="thumbnail"
            class="sc-fubCfw fKSrkK"
            src="https://i.ytimg.com/vi/NP1gAnbeNno/default.jpg"
          />
          <div
            class="sc-pFZIQ"
          >
            <span
              class="sc-jrAGrp gZeKgx"
            >
              Wizeline Querétaro | Mexico&#39;s New Knowledge Economy (We&#39;re hiring!)
            </span>
            <p
              class="sc-kEjbxe hAIuZE"
            >
              A small but mighty (and growing) team, the Queretaro crew has taken ownership of growing the office and brand, speaking at university events, hosting tech ...
            </p>
          </div>
        </div>
        <div
          class="sc-iBPRYJ dmMilx"
        >
          <img
            alt="thumbnail"
            class="sc-fubCfw fKSrkK"
            src="https://i.ytimg.com/vi/F6Krwu6lUc8/default.jpg"
          />
          <div
            class="sc-pFZIQ"
          >
            <span
              class="sc-jrAGrp gZeKgx"
            >
              Miriam Godinez | Women in Leadership at Wizeline
            </span>
            <p
              class="sc-kEjbxe hAIuZE"
            >
              Science and technology always caught Miriam's attention. One of her ultimate goals as a Senior Engineer Manager and Lead from the Mobile Team at Wizeline ...
            </p>
          </div>
        </div>
        <div
          class="sc-iBPRYJ dmMilx"
        >
          <img
            alt="thumbnail"
            class="sc-fubCfw fKSrkK"
            src="https://i.ytimg.com/vi/RFq7gfvhtCk/default.jpg"
          />
          <div
            class="sc-pFZIQ"
          >
            <span
              class="sc-jrAGrp gZeKgx"
            >
              Welcome Back to Wizeline Vietnam | Extended Version
            </span>
            <p
              class="sc-kEjbxe hAIuZE"
            >
              Thanks to swift government action, the COVID-19 situation in Vietnam has reached a point where businesses are able to return to work and reopen offices.
            </p>
          </div>
        </div>
        <div
          class="sc-iBPRYJ dmMilx"
        >
          <img
            alt="thumbnail"
            class="sc-fubCfw fKSrkK"
            src="https://i.ytimg.com/vi/E1Vq_A3WKK8/default.jpg"
          />
          <div
            class="sc-pFZIQ"
          >
            <span
              class="sc-jrAGrp gZeKgx"
            >
              How does Wizeline work?
            </span>
            <p
              class="sc-kEjbxe hAIuZE"
            >
              Wizeline builds teams with a mix of technical and non-technical talent to deliver better products, faster. Learn more about our consulting services: ...
            </p>
          </div>
        </div>
        <div
          class="sc-iBPRYJ dmMilx"
        >
          <img
            alt="thumbnail"
            class="sc-fubCfw fKSrkK"
            src="https://i.ytimg.com/vi/ZmkslANDz0Q/default.jpg"
          />
          <div
            class="sc-pFZIQ"
          >
            <span
              class="sc-jrAGrp gZeKgx"
            >
              12 Wishes from Wizeline | Happy Holidays 2019
            </span>
            <p
              class="sc-kEjbxe hAIuZE"
            />
          </div>
        </div>
        <div
          class="sc-iBPRYJ dmMilx"
        >
          <img
            alt="thumbnail"
            class="sc-fubCfw fKSrkK"
            src="https://i.ytimg.com/vi/Nss3EmTDD3s/default.jpg"
          />
          <div
            class="sc-pFZIQ"
          >
            <span
              class="sc-jrAGrp gZeKgx"
            >
              Why Wizeline?
            </span>
            <p
              class="sc-kEjbxe hAIuZE"
            >
              Hear from our employees directly about what excites them about their roles here at Wizeline. Wizeline wants to hire the best and the brightest to accelerate their ...
            </p>
          </div>
        </div>
        )
      </div>
    </div>
  </main>
</div>
`;
